### PR TITLE
docs: Added Open Api Docs for step-template-controller, workflow-controller, workflow-template-controller, and key-pair-controller

### PR DESCRIPTION
**Pull request by nguyen102**
Issue #, if available:

Description of changes:
chore: Added Open Api Docs for step-template-controller, workflow-controller, workflow-template-controller, and key-pair-controller

Validated API using https://editor.swagger.io/

Also test that import into POSTMAN worked correctly.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Bingjiling — COMMENTED** [review]
Looks good! Just a few minor description updates.

File: openapi.yaml [hunk 1]
```diff
@@ -950,6 +958,688 @@ paths:
...
+          schema:
+            $ref: '#/components/schemas/StepTemplateId'
+        - name: versionId
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/VersionId'
+      responses:
+        '200':
+          description: 'Return array of all versions of the requested step template'
```
[the remark this answers is not in the document]
Comment (nguyen102) [reply openapi.yaml:1033]: Good catch. Updating to
`Return requested step template`

File: openapi.yaml [hunk 2]
```diff
@@ -950,6 +958,688 @@ paths:
...
+          description: 'Return array of workflow drafts'
+    post:
+      tags:
+        - admins
+        - Workflows
+      summary: Create a draft workflow
+      operationId: createDraftWorkflow
+      requestBody:
+        required: true
+        description: ''
```
[the remark this answers is not in the document]
Comment (nguyen102) [reply openapi.yaml:1195]: Good catch. Removing this empty description.

File: openapi.yaml [hunk 3]
```diff
@@ -950,6 +958,688 @@ paths:
...
+      operationId: editDraftWorkflow
+      parameters:
+        - name: workflowId
+          in: path
+          required: true
+          schema:
+            $ref: '#/components/schemas/WorkflowId'
+      requestBody:
+        required: true
+        description: ''
```
[the remark this answers is not in the document]
Comment (nguyen102) [reply openapi.yaml:1218]: Good catch. Removing this empty description.

**Review by Bingjiling — DISMISSED** [review]
LGTM!

**Review by Bingjiling — APPROVED** [review]
LGTM

**Pull request merged**